src/content/developers/docs/oracles/index.md

```diff
@@ -422,29 +194,239 @@ contract Oracle {
 }
 ```
```

Comment:
typos in code snippet (line 125 and 126) lenght --> length

Reply:
Fixed this now.